voice/voice.go

```diff
@@ -4,11 +4,16 @@
 // This package abstracts the subpackage voice/voicesession and voice/udp.
 package voice
 
-import "github.com/diamondburned/arikawa/v2/gateway"
+import (
+	"github.com/diamondburned/arikawa/v2/gateway"
+	"github.com/diamondburned/arikawa/v2/gateway/shard"
+)
 
 // AddIntents adds the needed voice intents into gw. Bots should always call
 // this before Open if voice is required.
-func AddIntents(gw *gateway.Gateway) {
-	gw.AddIntents(gateway.IntentGuilds)
-	gw.AddIntents(gateway.IntentGuildVoiceStates)
+func AddIntents(m *shard.Manager) {
```

Comment:
I think this should be split into a version for `Gateway` and another for the shard manager, just to make things just as easy as before for people implementing their own shard manager.

Reply:
I think instead of having two methods, one for gateway and one for shard.Manager, it might be simpler to just have an expose `Intents` constant, that users add themselves.

Reply:
That works as well.